### PR TITLE
Added sdk.location.is method tests

### PR DESCRIPTION
**Pull request by MarynaG**
# Purpose of PR

Added tests, which verifies sdk.location.is method is true for all extension types.

### CODE REVIEW

File: test/cypress/integration/dialog-extension.spec.js [hunk 1]
```diff
@@ -69,6 +69,12 @@ context('Dialog extension', () => {
     })
   })
 
+  it('verifies sdk.location.is dialog', () => {
+    cy.getSdk(iframeDialogSelector).then(sdk => {
+      expect(sdk.location.is('dialog')).to.equal(true)
```
Comment (jelz) [inline test/cypress/integration/dialog-extension.spec.js]: let's test it's `false` for other values

Comment (davidfateh) [reply test/cypress/integration/dialog-extension.spec.js]: Should your suggestion also apply to the other tests below?

Comment (jelz) [reply test/cypress/integration/dialog-extension.spec.js]: https://github.com/contentful/ui-extensions-sdk/pull/238#pullrequestreview-278198669 ;)

Comment (MarynaG) [reply test/cypress/integration/dialog-extension.spec.js]: done, thanks!

**Comment by jelz** [comment]
What @davidfateh said in https://github.com/contentful/ui-extensions-sdk/pull/238#pullrequestreview-278246992

we should define a list of valid values:

```js
const VALID_LOCATIONS = ['dialog', 'entry-field', 'entry-sidebar' /* ... */];
```

and prepare an assertion function as proposed by David.

I'd also test for some input which is not on a list of valid locations:

```js
expect(sdk.location.is('BOOOOOM!')).to.equal(false)
```

File: test/cypress/utils/verify-location.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import locations from '../../../lib/locations'
+
+export function verifyLocation(sdk, location) {
+  expect(sdk.location.is(location)).to.equal(true)
+  let invalidLocations = Object.values(locations)
+  invalidLocations = invalidLocations.filter(item => item !== location)
+  invalidLocations.push('test-value')
+  console.log(invalidLocations)
```
Comment (davidfateh) [inline test/cypress/utils/verify-location.js]: Remove.
```suggestion

```

File: test/cypress/utils/verify-location.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+  invalidLocations.every(item => expect(sdk.location.is(item)).to.equal(false))
```
Comment (davidfateh) [inline test/cypress/utils/verify-location.js]: Minor, but I think a `forEach` would make more sense here as we don't use the return value of this code.
```suggestion
  invalidLocations.forEach(item => expect(sdk.location.is(item)).to.equal(false))
```

File: test/cypress/utils/verify-location.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import locations from '../../../lib/locations'
+
+export function verifyLocation(sdk, location) {
+  expect(sdk.location.is(location)).to.equal(true)
+  let invalidLocations = Object.values(locations)
+  invalidLocations = invalidLocations.filter(item => item !== location)
+  invalidLocations.push('test-value')
```
Comment (jelz) [inline test/cypress/utils/verify-location.js]: Let's push `invalid-location` so it's more obvious

Comment (MarynaG) [reply test/cypress/utils/verify-location.js]: fixed.

**Pull request merged**